styles.css

```diff
@@ -0,0 +1,406 @@
+@media only screen and (max-width: 765px) {
+  body {
+    margin: 0;
+    background-color: black;
```

Comment:
Kindly note that it's not the whole document that has a black background. I suggest you add the background color to only the sections that need it, that is the `headline` and `about`

This are examples of parts of the design that have a different background:
<img width="375" alt="Screenshot 2022-12-02 at 14 32 33" src="https://user-images.githubusercontent.com/48546992/205284209-8721d093-227f-43fb-b6ba-06842830beee.png">